CMakeLists.txt

```diff
@@ -1612,6 +1612,10 @@ if (BUILD_TESTS)
   include(CTest)
 endif (BUILD_TESTS)
 
+if (AARCH64)
+  option(BUILD_TESTS_SVE "Build tests with SVE enabled and high optimization (-O3)" OFF)
```

Comment:
I would expect the tests to be built at the same optimization/debug level as the rest of the project: should the tests be O3 for a debug build?  I would not expect that.

Reply:
In order to maximise SVE test coverage, we need to enable autovectorisation in the compiler. Without `-O3`, there will be very few SVE instruction sequences and the value of the functional tests greatly diminished.